### ISSUE TITLE
Close port 443 again

### ISSUE THREAD

**Issue by jakobbraun** [exasol/terraform-aws-exasol-test-setup#8]
We had to open port 443 so that the terraform-aws-exasol module can check if the startup has finished. However, this opens an attack vector. So after https://github.com/exasol/cloudformation-aws-exasol/issues/2 and by that https://github.com/exasol/terraform-aws-exasol/issues/22  is fixed, we can close that port again.

**Comment by ckunki** [comment]
Current ticket is blocked by https://github.com/exasol/terraform-aws-exasol/issues/22
which in turn is blocked by https://github.com/exasol/cloudformation-aws-exasol/issues/2
while the repo `cloudformation-aws-exasol` is currently archived, i.e. maybe noone is working on it

**Comment by ckunki** [comment]
Closing current ticket as "won't fix".
